### PR TITLE
MM-30522: Add DNS grafana dashboards

### PR DESCRIPTION
**Pull request by stafot**
Also changed order to be alphabetical for readability purposes

#### Summary

Adds Grafana dashboards for DNS monitoring. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-30522

### CODE REVIEW

File: terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf [hunk 1]
```diff
@@ -139,3 +137,31 @@ resource "kubernetes_config_map" "mattermost_performance_monitoring_per_cluster"
...
+      cloud_dashboards = "skydns"
+    }
+  }
+
+  data = {
+    "coredns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
+  }
+}
+
+resource "kubernetes_config_map" "uptime" {
```
Comment (angeloskyratzakos) [inline terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:155]: I would suggest to follow the pattern
```suggestion
resource "kubernetes_config_map" "uptime_view" {
```

Comment (stafot) [reply terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:155]: @angeloskyratzakos Agree. Need to do follow up changes on other places too, if we all agree  on this convention.

**Review by angeloskyratzakos — APPROVED** [review]

File: terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf (hunk 1 above)
Comment (angeloskyratzakos) [reply terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:155]: not talking about naming convention in general, but on the specific resource, lines +157 to +165

File: terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf [hunk 2]
```diff
@@ -139,3 +137,31 @@ resource "kubernetes_config_map" "mattermost_performance_monitoring_per_cluster"
...
+  metadata {
+    name      = "skydns-dashboard"
+    namespace = "monitoring"
+    labels = {
+      cloud_dashboards = "skydns"
+    }
+  }
+
+  data = {
+    "coredns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
```
Comment (angeloskyratzakos) [inline terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:151]: this one as well needs to be updated
```suggestion
    "skydns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
```

Comment (stafot) [reply terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:151]: Oops, typo! Nice catch @angeloskyratzakos !!!

File: terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf [hunk 3]
```diff
@@ -1,114 +1,112 @@
...
     namespace = "monitoring"
     labels = {
-      cloud_dashboards = "account_view"
+      cloud_dashboards = "account_monitoring"
     }
   }
 
   data = {
-    "account_view.json" = "${file("../../../../grafana-dashboards/account_view.json")}"
+    "acoount_monitoring.json" = "${file("../../../../grafana-dashboards/account_monitoring.json")}"
```
Comment (stylianosrigas) [inline terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:11]: @stafot type here

**Review by stylianosrigas — APPROVED** [review]
Appart from a typo, looks good.

File: terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf (hunk 3 above)
Comment (stafot) [reply terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:11]: it just changed position @stylianosrigas.  Account view is below.

Comment (angeloskyratzakos) [reply terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf:11]:
```suggestion
    "account_monitoring.json" = "${file("../../../../grafana-dashboards/account_monitoring.json")}"
```

**Review by angeloskyratzakos — APPROVED** [review]

**Pull request merged**